src/cruds/device.py

```diff
@@ -246,6 +246,48 @@ async def get_historical_device_data_alarm(db: AsyncSession, device_id: str) ->
     return result
 
 
+async def get_device_data_notifications(db: AsyncSession, device_id: str) -> List[device_schema.DeviceNotificationData]:
+    """
+    Get notifications data from db
+
+    Args:
+        db (AsyncSession): AsyncSession
+        device_id (str): Device id
+
+    Returns:
+        [device.schema.DeviceNotificationData]: List of device historical notification data
+    """
+    stmt = """
+        SELECT
+            CONTENT_TYPE,
+            CONTENT,
+            IS_READ,
+            TO_CHAR(CREATED_AT, 'YYYY/MM/DD') AS DATE
+        FROM NOTIFICATIONS
+        WHERE
+            DEVICE_ID = :device_id
```

Comment:
Change the WHERE condition to below.
This will select all device notification user has.
```sql
WHERE
    DEVICE_ID IN (
        SELECT
            ID
        FROM DEVICES
        WHERE
            USER_ID = :user_id
    )
```